sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -149,6 +151,9 @@ def _point_to_envelope(
         resource: Optional[Resource] = None,
         scope: Optional[InstrumentationScope] = None,
     ) -> Optional[TelemetryItem]:
+        # When Metrics to Log Analytics is disabled, only send Standard metrics and _OTELRESOURCE_
+        if not self._metrics_to_log_analytics and name not in _AUTOCOLLECTED_INSTRUMENT_NAMES:
```

Comment:
It feels a little strange to have this kind of logic in the metric to envelope conversion, why not adding this whenever the metric exporter is being created? you can just never add it entirely when the env variable is present and the value is false

Reply:
Talked to Jeremy, I thought all metrics should be turned off in this case so having the exporter being instantiated was not correct, but this is not correct, only specific kind of metrics are disabled in this case, so it makes sense to have this logic here.